addon/components/single-event.js

```diff
@@ -168,4 +177,34 @@ export default Component.extend({
     const daysSinceLastUpdate = today.diff(lastModifiedDate, 'days');
     return daysSinceLastUpdate < 6;
   }),
+
+  /**
+   * Callback function for <code>Array.sort()<code>.
+   * Compares two given Objects by their position property (in ascending order), and then by id (descending).
+   *
+   * @method positionSortingCallback
+   * @param {Object} obj1
+   * @param {Object} obj2
+   * @return {Number}
+   */
+  positionSortingCallback(obj1, obj2) {
```

Comment:
Why not use the `sortable-by-position` mixin here?

Reply:
`sortable-by-position` mixin assumes Ember objects as input. we're operating on plain on javascript objects here, so `get()` won't work here. so i re-implemented that sorting callback to work with POJsOs.

Reply:
Ah ok. Thanks.